content/docs/pipelines/sdk/sdk-overview.md

```diff
@@ -117,6 +117,34 @@ The Kubeflow Pipelines SDK includes the following packages:
   Kubeflow Pipelines. Examples include utility functions for on premises,
   Google Cloud Platform (GCP), Amazon Web Services (AWS), and Microsoft Azure.
 
+* [KFP diagnose_me modules](https://github.com/kubeflow/pipelines/tree/master/sdk/python/kfp/cli/diagnose_me)
+  These modules are part of the KFP CLI tool that are designed to help with environment diagnostic tasks such as 
+  ensuring required kubernetes secrets are set properly. 
+ 
+  * `kfp.cli.diagnose_me.dev_env` collects developer env related environment configurations such as python version.
+  * `kfp.cli.diagnose_me.kubernetes_cluster` collects Kubernetes cluster related configurations such as secrets.
+  * `kfp.cli.diagnose_me.gcp` collects Google Cloud Platform related environment configurations.
+ 
+## KFP CLI Tool 
+Kubeflow pipelines has a CLI tool that enables a subset of the sdk functionality directly from a command line in addition to environment diagnostic features. In particular the CLI tool provides the following features: 
+
+* `kfp diagnose_me` runs environment diagnostic with specified parameters.
```

Comment:
Suggested revisions - User input format, moving GCP specific setting to the end, calling out Project-ID as GCP specific:

* `kfp diagnose_me` runs environment diagnostic with specified parameters.
  * `--json` - Indicates that this command must return its results as JSON. Otherwise, results are returned in human readable format.
  * `--namespace <NAMESPACE>` - Specifies the Kubernetes namespace to use. **all-namespaces** is the default value.
  * `--project-id <PROJECT-ID>` - For GCP deployments, this value specifies the GCP project to use. If this value is not specified, the environment default is used.

Reply:
Got it, that makes sense.